packages/clay-navigation/src/BreadcrumbEllipsis.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayButton from '@clayui/button';
+import ClayDropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+import {IBreadcrumbItem} from './BreadcrumbItem';
+
+export interface IBreadcrumbEllipsisProps {
+	items?: Array<IBreadcrumbItem>;
+
```

Comment:
no need for line breaks here unless you are adding jsdocs

Reply:
Solved!